### PR TITLE
[TDP-138] Adding data migration to load initial model data

### PR DESCRIPTION
**Pull request by nbucknor**
Created a data migration to load the tdp_initial_data fixture.

@atuggle @chosak

### CODE REVIEW

**Comment by nbucknor** [comment]
@dcmouyard @Scotchester @chosak could one of you code review and merge this? Unfortunately, @atuggle  is unavailable at the moment.

File: teachers_digital_platform/migrations/0010_load_initial_data.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+from django.core import serializers
+
+fixture_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '../fixtures'))
+fixture_filename = 'tdp_initial_data.json'
+
+
+def load_fixture(apps, schema_editor):
+    fixture_file = os.path.join(fixture_dir, fixture_filename)
+
+    fixture = open(fixture_file, 'rb')
```
Comment (chosak) [inline teachers_digital_platform/migrations/0010_load_initial_data.py]: MInor suggestion: use a context manager (`with open(fixture_file, 'rb') as fixture:`) here instead. This avoids the need for the `close` and properly closes the file should something go wrong. See [the docs](https://docs.python.org/2/tutorial/inputoutput.html#methods-of-file-objects) (look down for "good practice").

File: teachers_digital_platform/migrations/0010_load_initial_data.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import os
+from django.db import migrations, models
+
+from sys import path
```
Comment (chosak) [inline teachers_digital_platform/migrations/0010_load_initial_data.py]: Unnecessary import.

Comment (nbucknor) [reply teachers_digital_platform/migrations/0010_load_initial_data.py]: I've removed "from sys import path"

**Comment by nbucknor** [comment]
@dcmouyard could you merge this into master?

**Comment by Scotchester** [comment]
@nbucknor Are you not able to merge? It looks like you're already on the team that affords write privileges here.

**Comment by nbucknor** [comment]
@Scotchester I can merge, but was informed it's not best practice to merge your own pull requests.

**Comment by Scotchester** [comment]
Our standard practice is to merge your own PRs after receiving at least one approval, unless someone else has asked for changes or more time to review.

**Pull request merged**